docs/en/concepts-and-designs/kubesphere-logging.md

```diff
@@ -0,0 +1,46 @@
+# KubeSphere Logging
+
+## Overview
+
+The KubeSphere logging system comprises many components that work together to achieve centralized log management. Once you enable the logging module during installation, a few components will be deployed or created within the namespace `kubesphere-logging-system`:
+
+- Fluent Bit: log collector deployed on each node.
+- Fluent-bit Operator: automates tasks for operating Fluent bit instances.
+- Logsidecar Injector: injects sidecar into pods for streaming on-disk files to stdout.
+- Elasticsearch Cluster: log storage backend.
+- Curator: performs stale log cleanup.
+
+The diagram below presents the overall logging architecture:
+
+![](../../images/kubesphere-logging-overview.png)
+
+The following sections will detail the design and functionality of each component. 
+
+## Log Collection and Forward
+
+There are two types of logs produced by containers:
+
+- stdout and stderr: written to `.log` files in the host's directory `/var/log/containers` by default.
+- application logs: files on the pod's volume, file locations may vary. 
+
+Fluent Bit serves as the logging agent. It can access host's container log directory and streams log files to stdout. In KubeSphere, we forked and modified the origin Fluent Bit project fluent/fluent-bit. The difference between fluent/fluent-bit and [kubesphere/fluent-bit](https://github.com/kubesphere/fluent-bit) lies that we add a reload interface to support dynamic configuration (see issue [#365](https://github.com/fluent/fluent-bit/issues/365)).
+ 
+As shown in the diagram below, Fluent-bit controller implements the reload interface and is responsible for the life cycle of fluent-bit process.
+
+![](../../images/kubesphere-logging-fluentbit.png) 
+
+The approach to collect container logs on a pod's volume is using a log forwarder sidecar. The sidecar container forwards on-disk application logs to its own stdout. In this way, KubeSphere manages log collection in a unified manner. Besides, [Logsidecar Injector](https://github.com/kubesphere/logsidecar-injector) comes to automate sidecar injection. It takes advantages of `MutatingAdmissionWebhook` to control log forwarder sidecar injection during pod creation.
```

Comment:
a unified => an unified

Reply:
seems "a" is correct

https://www.merriam-webster.com/dictionary/unified

Reply:
:) You're right, U is pronounced as /ju:/ here